tests/Console/Service/CacheOpenAPIProcessorsTest.php

```diff
@@ -25,6 +25,8 @@
 use PHPUnit\Framework\TestCase;
 use Psr\Log\LoggerInterface;
 
+use function PHPUnit\Framework\assertDirectoryExists;
```

Comment:
Does this line need to be here?

Reply:
nope... PHPStorm must have added that when I used the function instead of the self:: version by mistake